index.d.ts

```diff
@@ -1,32 +1,37 @@
-interface BaseSignOptions {
-  app: string;
-  identity?: string;
-  platform?: string;
-  keychain?: string;
-}
+declare module "electron-osx-sign" {
```

Comment:
Otherwise typing is not used if `module: es2015` Why `es2015` module is used for node projects? Because it is transpiled by babel in the end :)